### DISCUSSION TITLE
Looking Ahead – Industry Trends and New Zigbee Features (Zigbee Revision 23)

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#1081] [General]
FYI, CSA (formerly Zigbee Alliance) posted a blog about Industry Trends and New Zigbee Features related to Zigbee Revision 23 (R23):

https://csa-iot.org/newsroom/looking-ahead-industry-trends-and-new-zigbee-features/

Zigbee 3.0 Revision 23 (R23), the next version of the Zigbee PRO standard, expands on existing security by adding new security features: 

* Dynamic Link Key (DLK) joining: Using elliptic curve Diffie-Hellman key negotiation to establish keys and authenticate devices before joining the network 
* Device interview: Using DLK, information is requested to identify the device before accepting it into the network 
* Frame counter synchronization to provide advanced message replay protection
* Network parameter security with PANID and channel management

In addition, it brings up the awareness of the Zigbee Green Power (ZGP) technology, which I believe is now the only major feature that is missing from zigpy(?), at least if do not count Zigbee Smart Energy (ZSE) profile which is currently not part of Zigbee 3.0 specification.

https://github.com/zigpy/zigpy/issues/341

It also mentions Zigbee’s newest product feature, Zigbee Direct, which posted as a separate discussion since not directly related to zigpy:

https://github.com/zigpy/zigpy/discussions/1080

**Comment by MattWestb** [comment]
Recommended reading the new Matter specification 1.0:
`22-27349-001_Matter-1.0-Core-Specification76.pdf`
`22-27350-001_Matter-1.0-Application-Cluster-Specification15.pdf`
`22-27351-001_Matter-1.0-Device-Library-Specification39.pdf`
from https://csa-iot.org/developer-resource/specifications-download-request/

Its very "Zigbee" (ZCL) so very likely HA is getting one upper layer for Matter and Zigbee / thread is being one underlying transport layer.

Also interesting is DALI is being one part of the Matter but i have not reading how it shall working bridging between the different underlying platforms.

I think Zigbee R23 is being the release of Zigbee 3 upgrade for Matter and is using all new futures for commissioning (Zigbee direct) for working with Matter and also likely support multiple protocols like Zigbee / thread and BT on the same chip (mode switching so can adding one light SOC as Zigbee if having one Zigbee mesh, BT or thread if having one OTBR up and running with the same chip and firmware on it).
